lib/ohai/plugins/php.rb

```diff
@@ -1,8 +1,8 @@
 #
-# Author:: Doug MacEachern <dougm@vmware.com>
-# Author:: Tim Smith <tim@cozy.co>
-# Copyright:: Copyright (c) 2009 VMware, Inc.
-# Copyright:: Copyright (c) 2014 Cozy Services, Ltd.
```

Comment:
It seems incorrect to remove the copyrights of my former employers on these contributions

Reply:
@johnmccrae  lets hold on this change. @tas50  i will get back on it.

Reply:
yes but the copyright notice is the concern here. Let me discuss with legal and get back.